programmers/난이도별/level02.영어_끝말잇기/SSSOy.java

```diff
@@ -0,0 +1,30 @@
+import java.util.*;
+
+class Solution {
+    public int[] solution(int n, String[] words) {
+        int[] answer = new int[2];
+        List<String> words_list = new ArrayList();
+        int turn_cnt = 1;
+        int player_num = 2;
+        
+        words_list.add(words[0]);
+        
+        for(int i = 1; i < words.length; i++) {
+            if(words_list.indexOf(words[i]) != -1 || words[i].charAt(0) != words[i - 1].charAt(words[i - 1].length() - 1)) {
```

Comment:
```suggestion
            String nowWord = words[i], beforeWord = words[i - 1];
            if(!words_list.contains(nowWord) || nowWord.charAt(0) != beforeWord.charAt(beforeWord.length() - 1)) {
```
- index로 구분하기보단 변수로 정의해서 구분해주세요!
- 중복 검사는 `indexOf`보단 `contains`를 사용하면 더 편합니다!

Reply:
리뷰 감사합니다!! contains 함수를 사용하니까 코드가 더 깔끔해진 것 같아요!ㅎㅎ